sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
+        Which value(s) passed to ``transform()`` to be considered missing
+
+    handle_missing : string, default 'ignore'
+        How to handle missing values passed to ``transform()``
```

Comment:
is it only for `transform`?

Reply:
Yes, an assumption I'm making is the data fit to the encoder will not have missing values. Is this incorrect?

With this version, if data with missing values is fit to the Encoder, it will be appended to the categories of the instance. Missing values would still get encoded appropriately in `transform`, but this means the transformed data will have one missing category. 

Should I expand this implementation to make `fit` tolerant to data with missing values?

Reply:
Yes, data passed to `fit` can contain NaNs. So fit and transform should be consistent with regard to this `handle_missing` keyword. Eg if it is set to 'ignore' (preserving them in the output), NaNs should not be added to the categories during fit.

Reply:
Okay, good to know. I'll make it consistent.